### DISCUSSION TITLE
Add waveform panel

### DISCUSSION THREAD

**Discussion by iris2jane** [aidenlx/media-extended#652] [Ideas]
Can the waveform panel area be added in the video and audio interface to facilitate sentence breaking or positioning

能否在视频和音频界面增加波形面板区域，便于快速断句或定位

![image](https://user-images.githubusercontent.com/36763930/135199034-bbdb8ff8-2d0f-4d2c-955f-566d5f4e73ab.png)

**Comment by aidenlx** [comment]
https://collab-project.github.io/videojs-wavesurfer/#/examples
https://wavesurfer-js.org/example/video-annotation/index.html
